source/atlas-vector-search.txt

```diff
@@ -0,0 +1,178 @@
+Atlas. Vector search allows you to query your data based on semantic meaning
+rather than just keyword matches, which helps you retrieve more relevant search
+results. It enables your AI-powered applications to support use cases such as
+semantic search, hybrid search, and generative search, including
+Retrieval-Augmented Generation (RAG).
+
+To learn more about {+vector-search+}, see the :atlas:`{+vector-search+}
+</atlas-vector-search/vector-search-overview/>` guides in the MongoDB Atlas
+documentation.
+
+.. important:: Feature Compatibility
+
+   To learn what versions of MongoDB Atlas support this feature, see
+   :atlas:`Limitations </atlas-vector-search/vector-search-stage/#limitations>`
+   in the MongoDB Atlas documentation.
+
+Perform a Vector Search
+-----------------------
+
+To use this feature, you must create a vector search index and index your
+vector embeddings. To learn about how to programmatically create a
+vector search index, see the :atlas:`Create an Atlas Vector Search Index
+</atlas-vector-search/vector-search-type/#create-an-atlas-vector-search-index>` section in the
+Atlas documentation. To learn more about vector embeddings, see
+:atlas:`How to Create Vector Embeddings
+</atlas-vector-search/create-embeddings/>` in the Atlas documentation.
+
+After you create a vector search index on your vector embeddings, you
+can reference this index in your pipeline stage, as shown in the
+following example.
+
+Sample Data
+~~~~~~~~~~~
+
+The example on this page shows how to build an aggregation pipeline that uses the
+``$vectorSearch`` stage to perform a vector search on the
+``sample_mflix.embedded_movies`` collection in the :atlas:`Atlas sample datasets
+</sample-data>`.
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Vector Search Example
+~~~~~~~~~~~~~~~~~~~~~
+
+You can perform a vector search query by using the ``$vectorSearch`` stage
+in an :ref:`aggregation pipeline <node-aggregation>`. To perform a vector
+search on a collection, you must first have a collection with a field that contains
+vector embeddings and a vector search index that covers that field. 
+
+In the following example, the aggregation pipeline searches the ``plot``  field of each
+document in the collection for text semantically related to the term "time travel". The
+``queryVector`` field in the ``$vectorSearch`` pipeline is the vector representation of
+your query.
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/aggregation/atlas-vector-search-example.js
+      :language: javascript
+      :dedent:
+
+   .. output:: 
+      :language: json
+      :visible: false
+
+      {
+      plot: 'A reporter, learning of time travelers visiting 20th century disasters, tries to change the history they know by averting upcoming disasters.',
+      title: 'Thrill Seekers',
+      score: 0.9259490966796875
+      }
```

Comment:
```suggestion
      {
        plot: 'A reporter, learning of time travelers visiting 20th century disasters, tries to change the history they know by averting upcoming disasters.',
        title: 'Thrill Seekers',
        score: 0.9259490966796875
      }
```

I assume this output is being copied from an actual run, something happened to the usual indentation here